dex/src/test/scala/com/wavesplatform/dex/MatcherSpecBase.scala

```diff
@@ -50,9 +52,21 @@ trait MatcherSpecBase
 
   implicit override def patienceConfig: PatienceConfig = PatienceConfig(2.seconds, 5.millis)
 
-  implicit protected val wsErrorDiff: Derived[Diff[WsError]] = Derived(Diff.gen[WsError].ignore[WsError, Long](_.timestamp))
+  implicit protected def wsErrorDiff: Derived[Diff[WsError]] = Derived(Diff.gen[WsError].ignore[WsError, Long](_.timestamp))
```

Comment:
Wondering, why do we use `def` here? `val` seems fine. Or not? :)

Reply:
discussed privately =)